PredixMobileUIKit/PredixBarChartView+IBInspectable.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  PredixBarChartView+IBInspectable.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 12/10/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+
+extension PredixBarChartView {
+
+    /// :nodoc:
+    open override func prepareForInterfaceBuilder() {
+        super.prepareForInterfaceBuilder()
+
+        let months = ["Jan", "Feb", "Mar", "Apr", "May"]
+        let unitsSold = [20.0, 4.0, 2.0, 5.0, 12.0]
+        let unitsBought = [10.0, 2.0, 6.0, 3.0, 5.0]
+
+        let unitsSoldBar = Bar(unitsSold, label: "Units Sold", colors: [NSUIColor.gray])
+        let unitsBoughtBar = Bar(unitsBought, label: "Units Bought", colors: [NSUIColor.orange])
+        create(xAxisValues: months, bars: [unitsSoldBar, unitsBoughtBar], stackBars: true, showWithDefaultAnimation: false)
+    }
+
+    // MARK: - IBInspectable properties
+
+    /// the chart embedded label text
+    @IBInspectable
+    internal var labelText: String? {
+        get {
+            return chartDescription?.text
+        }
+        set(newValue) {
+            chartDescription?.text = newValue
+        }
+    }
+
+    // enable or disable the chart message embedded label text
+    @IBInspectable
+    internal var labelEnabled: Bool {
+        get {
+            return chartDescription?.enabled ?? false
+        }
+        set(newValue) {
+            chartDescription?.enabled = newValue
+        }
+    }
+
+    /// the chart message when there's not data
+    @IBInspectable
+    internal var noDataMessage: String {
+        get {
+            return noDataText
+        }
+        set(newValue) {
+            noDataText = newValue
+        }
+    }
+
+    /// indicates whether the legend chart should be aligned on the left *default:* true
```

Comment:
These code comment lines with *default* look awkward in Xcode's help window. Recommend the *default* be on a new line.